src/api/app/models/review.rb

```diff
@@ -28,6 +38,20 @@ def state
     read_attribute(:state).to_sym
   end
 
+  def accepted_at
+    if assigned_to && assigned_to.state == :accepted
+      return assigned_to.updated_at
+    elsif state == :accepted
+      return updated_at
+    else
+      return nil
+    end
```

Comment:
A ruby method will return nil by default if theres nothing return from it so you don't need the `else` claus. Also the ruby style guide says to prefer not to use `return` if you dont have to so you can do this. That said I actually do like using `return` personally...

```ruby
if assigned_to && assigned_to.state == :accepted
  assigned_to.updated_at
else state == :accepted
  updated_at
end
```